docs/en-US/guide/ssr.md

```diff
@@ -31,6 +31,20 @@ app.provide(ID_INJECTION_KEY, {
 })
 ```
 
+## Provide ZIndex
+
+When you using SSR for development, you may encounter hydration errors caused by `z-ndex`. In this case, we recommend injecting an initial value to avoid such errors.
```

Comment:
```suggestion
When you using SSR for development, you may encounter hydration errors caused by `z-index`. In this case, we recommend injecting an initial value to avoid such errors.
```